lua/damagelogs/client/init.lua

```diff
@@ -204,21 +204,21 @@ hook.Add("Think", "Think_Damagelog", function()
 end)
 
 function Damagelog:StrRole(role)
-    if not ROLES then
```

Comment:
I'm not really sure why this variable is in here.
Do you know where it comes from?
I can't find it defined anywhere in this addon, or the terrortown gamemode.

Reply:
ROLES used to be defined in TTT2, they renamed it to TTT2 so basically that is the old TTT2 detection and I had to go through and update that